lib/src/components/parallax_component.dart

```diff
@@ -93,7 +92,8 @@ class ParallaxComponent extends PositionComponent {
   /// used can also be passed in.
   /// If no image cache is set, the global flame cache is used.
   static Future<ParallaxComponent> load(
-    List<String> paths, {
+    List<String> paths,
+    Vector2 size, {
```

Comment:
Shouldn't `size` be optional here too? 
I feel like it would be cleaner in the component part of the Parallax that we relied on the initial `onGameResize` which is being called in `prepare`.

Reply:
Fair enough! Done